MAUI/DataForm/CustomDataFormItem.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: Custom DataFormItem in Maui DataForm control | Syncfusion
+description: Learn here all about Working with DataForm support in Syncfusion Maui DataForm (SfDataForm) control and more.
+platform: Maui
+control: SfDataForm
+documentation: UG
+---
+
+## Adding custom DataFormItems
+
+Support has been provided to generate custom DataFormItems for the defined business model using the [Items]`Link` property of the `SfDataForm` class. You need to set the [AutoGenerateItems]`Link` property to false to restrict the auto generation of DataFormItems. 
+
+{% tabs %}
+{% highlight xaml %}
+    <?xml version="1.0" encoding="utf-8" ?>
+    <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+                x:Class="GettingStarted.MainPage">
+        <ContentPage.BindingContext>
+            <local:ViewModel x:Name="bindingContext"/>
+        </ContentPage.BindingContext>
+
+        <ContentPage.Content>
+            <dataForm:SfDataForm x:Name="dataForm" DataObject="{Binding Details}"  AutoGenerateItems="false"> 
+                <dataForm:SfDataForm.Items> 
+                    <dataForm:DataFormTextItem FieldName="Name" /> 
+                    <dataForm:DataFormTextItem FieldName="Password" /> 
+                    <dataForm:DataFormTextItem FieldName="Address" /> 
+                    <dataForm:DataFormAutoCompleteItem FieldName="Countries"  ItemsSource = "{Binding CountryNames}"/> 
+                    <dataForm:DataFormDateItem FieldName="BirthTime" /> 
+                </dataForm:SfDataForm.Items> 
+            </dataForm:SfDataForm>
+        </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight MainPage.xaml.cs %}
+
+    ObservableCollection<DataFormViewItem> items = new ObservableCollection<DataFormViewItem>();
+    items.Add(new DataFormTextItem() { FieldName = "Name" });
+    items.Add(new DataFormPasswordItem() { FieldName = "Password"});
+    items.Add(new DataFormMultilineItem() { FieldName = "Address" });
+    items.Add(new DataFormAutoCompleteItem() { FieldName = "Countries", ItemsSource = this.GetItemSource("Countries")});
+    items.Add(new DataFormTimeItem() { FieldName = "BirthTime"}); 
+    dataForm.AutoGenerateItems = false; 
+    dataForm.Items = items; 
+
+    public class ViewModel
```

Comment:
change model

public class DataFormViewModel
    {
        public DataFormViewModel()
        {
            this.Detailsdetails = new DataFormModel();
            this.CountryNames = new List<string>
            {
                "United States",
                "United Kingdom",
                "France",
                "Belgium",
                "Germany"
            };
        }

        public ContactInfo ContactDetails { get; set; }

        public IList<string> CountryNames { get; set; }

    }

Reply:
code updated